wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt

```diff
@@ -128,7 +130,7 @@ fun MainView(
                                     )
                                 },
                                 onClick = {
-                                    onEntityClicked(favoriteEntityID)
+                                    onEntityClicked(favoriteEntityID, "unknown")
```

Comment:
Hmm so how is this going to work when a user adds a `lock` as a favorite when we don't have the proper state?

Reply:
Very good question. So if we don't have a state, there are three options:

1. Do nothing
2. Always lock
3. Always unlock

I think option 3 should be ruled out, since you don't want to accidentally unlock your front door. For the other two it is debatable. Right now, the default action is to lock (so also if the state is "unknown"). If you want we can update it to do nothing if the state does not have one of the two predefined values.

Reply:
Hmm this is a tough one, I think if we default to locking the lock that would be acceptable as we don't want to cause an accidental unlock :) I am also ok with doing nothing too.

Reply:
Hmm yeah either is fine for me :)

Is there any behavioral guidelines from home assistant? Or do you want to make the call?

Reply:
Hmm don't see anything mentioned in the docs: https://developers.home-assistant.io/docs/core/entity/lock/

We can wait for code review and see what @JBassett thinks we should do :)

Reply:
Locking is fine for now.  Once we cache the state in the DB we can use that  to determine state and what the next action should be.